src/View/Helper/FlashHelper.php

```diff
@@ -26,7 +26,16 @@ class FlashHelper extends Helper
 {
 
     /**
-     * Used to render the message set in FlashComponent::set()
+     * Default config for the helper.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'stackElement' => false
```

Comment:
My concern was that the setting could be enabled in the component, but not in the helper by accident causing the helper output to break.